.cargo/config.toml

```diff
@@ -20,7 +20,9 @@ rustflags = [
 [target.aarch64-apple-darwin]
 rustflags = [
   "-C",
-  "link-args=-fuse-ld=lld -weak_framework Metal -weak_framework MetalPerformanceShaders -weak_framework QuartzCore -weak_framework CoreGraphics",
+  "link-args=-weak_framework Metal -weak_framework MetalPerformanceShaders -weak_framework QuartzCore -weak_framework CoreGraphics",
+  "-C",
+  "prefer-dynamic",
```

Comment:
Can you explain why this is changing? Does it have impact on startup time performance?

Reply:
Oof this was an unrelated change I was playing around with, accidentally committed it